scenes/quests/lore_quests/quest_001/2_ink_combat/components/throwing_enemy/components/throwing_enemy.gd

```diff
@@ -156,7 +156,7 @@ func _process(_delta: float) -> void:
 			return
 		State.IDLE:
 			if animated_sprite_2d.animation not in [&"attack anticipation", &"attack"]:
```

Comment:
You have changed the name of `attack_anticipation` so I think you need to change this too.

I also noticed that the inkdrinkers do not reliably disappear when the level is solved.